src/Chapter2.hs

```diff
@@ -665,7 +710,7 @@ specify space-separated arguments. Instead of the "=" in the ordinary
 function body, you should write "->" and then you can use these
 arguments and all variables in scope inside the lambda-body.
 
-There are equal:
+These are equal:
```

Comment:
Btw, if you see some typos in our tasks, feel free to open PRs directly to our `master` branch in `learn4haskell` repo!

Reply:
:+1: